### DISCUSSION TITLE
Malloy an SQL alternative.

### DISCUSSION THREAD

**Discussion by ansarizafar** [databendlabs/databend#3407] [Ideas]
SQL is complete but ugly: everything is expressible, but nothing is reusable; simple ideas are complex to express; the language is verbose and lacks smart defaults. Malloy is immediately understandable by SQL users, and far easier to use and learn. It is usable in place of SQL to manipulate and explore data.

Please consider adding support for Malloy https://github.com/looker-open-source/malloy

**Comment by Xuanwo** [comment]
Sounds great.

Let's convert this issue to Discussions ideas so that we can come over this idea further.

ping @BohuTANG, can you help convert it?
